site/docs/v1/tech/apis/identity-providers/_samlv2-request-body.adoc

```diff
@@ -57,6 +73,20 @@ ifndef::samlv2_idp_initiated[]
 The SAML v2 login page of the identity provider. Must be a valid URL.
 endif::[]
 
+ifndef::samlv2_idp_initiated[]
+[field]#identityProvider.idpInitiatedConfiguration.enabled# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.43.0#::
+Determines if FusionAuth will accept IDP-initiated login requests from this SAMLv2 Identity Provider. A valid license is required to use this feature.
++
+In general FusionAuth does not recommend you utilize this type of login due to the inherent risks of accepting an unsolicited authentication response from a third-party identity provider. In a service provider initiated login, the SAML Authentication Response is returned to the service provider (FusionAuth) in response to an authentication request. This means FusionAuth can validate certain parameters and only accept a response for a request that FusionAuth initiated. In contrast, during an identity provider initiated login, the third-party identity provider sends FusionAuth a SAML Authentication Response indicating a user has been authenticated without any context. The problem is that FusionAuth did not initiate this so it is unexpected, this limits what can be verified and requires FusionAuth to allow a cross site request from this third party. If you must enable this feature to support a legacy integration or because the user experience is of the highest concern, then you may proceed.
```

Comment:
We may need some paragraph breaks so it is consumable -it looks like a wall of text when you render the UI.